documentation/source/ModelPredictions.md

```diff
@@ -0,0 +1,181 @@
+# Using Pretrained Models for Predictions 
+
+In this tutorial, we will demonstrate how to use the `model.predict()` method for object detection tasks.
+
+The model used in this tutorial is [YOLO-NAS](YoloNASQuickstart.md), pre-trained on the [COCO dataset](https://cocodataset.org/#home), which contains 80 object categories.
+
+
+**Warning**: If your model was trained on your own custom dataset, make sure to follow these [steps first](DetectionPredictionCustom.md).
+
+*Note that the  `model.predict()` method is currently only available for detection tasks.*
+
+
+## Detect Objects in Multiple Images
+
+#### Load the Model and Prepare the Images
+First, let's load the pre-trained `Yolo-NAS` model using the `models.get()` function and define a list of image paths or URLs that we want to process:
+
+```python
+from super_gradients.common.object_names import Models
+from super_gradients.training import models
+
+model = models.get(Models.YOLO_NAS_L, pretrained_weights="coco")
+```
+
+### Detect Objects in the Images
+The `model.predict()` method returns an `ImagesDetectionPrediction` object, which contains the detection results for each image.
+
+```python
+IMAGES = [
+    "path/to/local/image1.jpg",
+    "path/to/local/image2.jpg",
+    "https://example.com/image3.jpg",
+]
+
+images_predictions = model.predict(IMAGES)
+```
+You can use the default IoU and Confidence threshold or override them like this:
+
+```python
+images_predictions = model.predict(IMAGES, iou=0.5, conf=0.7)
+```
+- `iou`: IoU threshold for the non-maximum suppression (NMS) algorithm. If None, the default value associated with training is used.
```

Comment:
Thats (currently) not true, its defined by the model class.

Reply:
https://github.com/Deci-AI/super-gradients/commit/35af53371555093a4ab5aea8bb73120296bb8ee6